src/gui/tray/ActivityListModel.cpp

```diff
@@ -75,6 +76,13 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
         return QVariant();
 
     switch (role) {
+    case IsColoredIcon: {
+        if (a._fileAction == "file_created" || a._fileAction == "file_deleted") {
```

Comment:
@FlexW You can simply

`return (a._fileAction == "file_created" || a._fileAction == "file_deleted")`